<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewElement</class>
 <widget class="QDialog" name="NewElement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: rgb(186, 186, 186);</string>
  </property>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>340</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Retro Gaming</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Перон:</string>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>323</y>
     <width>401</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Serif</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	border : 4px solid white;
	border-bottom: 4px solid black;
	border-right: 4px solid black;
}
QPushButton:pressed{ 
	border : 4px solid black;
	border-bottom: 4px solid white;
	border-right: 4px solid white;
}</string>
   </property>
   <property name="text">
    <string>Додати</string>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>48</y>
     <width>3</width>
     <height>113</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_5">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>330</y>
     <width>3</width>
     <height>113</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>0</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Retro Gaming</family>
     <pointsize>19</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Новий елемент</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Serif</family>
     <pointsize>19</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">	border : 3px solid black;
	border-bottom: 3px solid white;
	border-right: 3px solid white;
	background-color:rgb(204, 204, 204);
font: 75 19pt &quot;MS Serif&quot;;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Retro Gaming</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>До міста:</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>48</y>
     <width>393</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditB">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>283</y>
     <width>351</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	border : 3px solid black;
	border-bottom: 3px solid white;
	border-right: 3px solid white;
	background-color:rgb(224, 224, 224);
font: 75 19pt &quot;MS Serif&quot;;</string>
   </property>
   <property name="text">
    <string>Москва</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Retro Gaming</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Дата</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Retro Gaming</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>З міста:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>60</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Retro Gaming</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Час</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditA">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>203</y>
     <width>351</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	border : 3px solid black;
	border-bottom: 3px solid white;
	border-right: 3px solid white;
	background-color:rgb(224, 224, 224);
font: 75 19pt &quot;MS Serif&quot;;</string>
   </property>
   <property name="text">
    <string>Харків</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>370</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	border : 3px solid black;
	border-bottom: 3px solid white;
	border-right: 3px solid white;
	background-color:rgb(204, 204, 204);
font: 75 19pt &quot;MS Serif&quot;;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>9</number>
   </property>
   <property name="value">
    <number>9</number>
   </property>
  </widget>
  <widget class="QTimeEdit" name="timeEdit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>100</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Serif</family>
     <pointsize>19</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">	border : 3px solid black;
	border-bottom: 3px solid white;
	border-right: 3px solid white;
	background-color:rgb(204, 204, 204);
font: 75 19pt &quot;MS Serif&quot;;
</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>160</y>
     <width>393</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
